File: lab2-semester1/1.cpp
//C++  Program to Calculate Area and Circumference of Circle

#include<iostream>
using namespace std;

int main(){
    float r, area, circum;
    float pi=3.14;
    
    cout<<"Enter lenght of radius"<<endl;
    cin>>r;
    
    area=pi*r*r;
    circum=2*pi*r;
    
      cout<<"\nArea of Circle = "<<area<<endl;
      cout<<"Circumference of Circle = "<<circum<<endl;
}

File: lab2-semester1/1.go
// Program to Calculate Area and Circumference of Circle

package main

import "fmt"

func main() {
    var r, area, circum float64
    const pi float64 = 3.14
    
    fmt.Print("Input lenght of radius: ")
    fmt.Scanln(&r)//input from user
    area=pi*r*r //formula to calculate area of circle
    circum=2*pi*r // formula for Circumference
    fmt.Println("Area: ", area)
    fmt.Println("Circumference: ", circum)
}
